pkgs/development/python-modules/numpy/default.nix

```diff
@@ -71,8 +74,6 @@ in buildPythonPackage rec {
     inherit blasImplementation cfg;
   };
 
-  doCheck = blasImplementation != "mkl";
```

Comment:
Numpy actualy passes all of its test cases with `mkl` as its `blas` implementation.